File: TrabajoFinalPrueba1/PegasusV1/PegasusV1/Entities/IntegrantesMaterias.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    [Table("INTEGRANTES_MATERIAS")]
    public class IntegrantesMaterias
    {
        public int Id { get; set; }

        [ForeignKey("Id_Materia")]
        public Materia? Materia { get; set; }
        public int? Id_Materia { get; set; }

        [ForeignKey("Id_Usuario")]
        public Usuario? Usuario { get; set; }
        public int? Id_Usuario { get; set; }


    }
}

File: pegasusapp/src/models/integrantesMaterias.ts
import { Materia } from './materia';
import { Usuario } from './usuario';

export interface IntegrantesMaterias {
  id: number;

  materia?: Materia | null;
  id_Materia?: number | null;

  usuario?: Usuario | null;
  id_Usuario?: number | null;
}
